data_registry/models.py

```diff
@@ -143,8 +144,8 @@ def complete(self):
 
 class CollectionQuerySet(models.QuerySet):
     def visible(self):
-        """Return a query set of public collections with active jobs."""
-        return self.filter(public=True, active_job__isnull=False)
+        """Return a query set of public collections with active jobs or a rationale for having no data."""
+        return self.filter(Q(public=True) & (Q(active_job__isnull=False) | ~Q(no_data_rationale__exact="")))
```

Comment:
`exact` is the default: https://docs.djangoproject.com/en/5.2/ref/models/querysets/#field-lookups

Chaining filters is a bit easier to read.

```suggestion
        return self.filter(public=True).exclude(active_job__isnull=True, no_data_rationale="")
```